components/mosart/bld/namelist_files/namelist_defaults_mosart.xml

```diff
@@ -32,4 +39,34 @@ for the CLM data in the CESM distribution
 <frivinp_rtm rof_grid="r05" >rof/mosart/MOSART_global_half_20180721a.nc</frivinp_rtm>
 <frivinp_rtm rof_grid="NLDAS" >rof/mosart/MOSART_NLDAS_8th.nc</frivinp_rtm>
 
+<!-- WRM -->
+
+<paraFile rof_grid="NLDAS" >rof/mosart/US_reservoir_8th_NLDAS3_c20161220_updated_20170314.nc</paraFile>
+<demandPath rof_grid="NLDAS" >/pic/scratch/vois549/wm_data/NLDAS2_GCAM_water_demand_</demandPath> 
```

Comment:
Why is this path hard-coded for PNNL machine? Also, the path should be relative to the inpudata dir for various machines.